scripts/roles.json

```diff
@@ -3062,15 +3305,18 @@
     "abstract": false,
     "accessibleNameRequired": false,
     "childrenPresentational": false,
-    "nameFrom": ["author"],
+    "nameFrom": ["author", "legend"],
```

Comment:
`author`, `name`, `legend`, `encapsulation`: for the enum.

Reply:
I don't understand what you mean. Latest editor's draft only lists these two values: https://rawgit.com/w3c/aria/1d856c402eafbe7823021ff4cb5ca44dc1a9edf7/#group

working draft only lists "author".

Reply:
> I don't understand what you mean. Latest editor's draft only lists these two values: https://rawgit.com/w3c/aria/1d856c402eafbe7823021ff4cb5ca44dc1a9edf7/#group
> 
> working draft only lists "author".

Sorry, this is confusing on my part. I meant only that these are the four values to include in the enum. There's nothing to change here.